sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -659,7 +757,8 @@ Duration getDefaultPollingInterval() {
         context = context == null ? Context.NONE : context;
         KeyRequestParameters parameters = new KeyRequestParameters()
             .setTags(keyProperties.getTags())
-            .setKeyAttributes(new KeyRequestAttributes(keyProperties));
+            .setKeyAttributes(new KeyRequestAttributes(keyProperties))
+            .setKeyReleasePolicy(keyProperties.getKeyReleasePolicy());
```

Comment:
Given you're copying this around and because `KeyProperties` needs it anyway, perhaps we should just define it on `KeyProperties`, which means `KeyVaultBundle.properties.releasePolicy` is where'd you get it.

Reply:
Agreed based on a separate conversation we had. Let's keep it only in `KeyProperties`.